drivers/input/touchscreen/synaptics_dsx_i2c.c

```diff
@@ -3809,11 +3857,14 @@ static void synaptics_rmi4_set_params(struct synaptics_rmi4_data *rmi4_data)
 	set_bit(KEY_HOMEPAGE, rmi4_data->input_dev->keybit);
 	set_bit(KEY_F3, rmi4_data->input_dev->keybit);
 	set_bit(KEY_DOUBLE_TAP, rmi4_data->input_dev->keybit);
-	set_bit(KEY_GESTURE_CIRCLE, rmi4_data->input_dev->keybit);
-	set_bit(KEY_GESTURE_SWIPE_DOWN, rmi4_data->input_dev->keybit);
-	set_bit(KEY_GESTURE_V, rmi4_data->input_dev->keybit);
-	set_bit(KEY_GESTURE_LTR, rmi4_data->input_dev->keybit);
-	set_bit(KEY_GESTURE_GTR, rmi4_data->input_dev->keybit);
+	set_bit(EV_MSC, rmi4_data->input_dev->evbit);
+	rmi4_data->input_dev->mscbit[0] = BIT_MASK(MSC_GESTURE);
```

Comment:
This looks a bit odd to me. Was expecting to see series of calls like

    set_bit(MSC_GESTURE_xxx, rmi4_data->input_dev->mscbit);